### DISCUSSION TITLE
PS/2 Model 70 ESDI Issues

### DISCUSSION THREAD

**Discussion by tifasoftware** [86Box/86Box#4411] [Q&A] [answered]
I am attempting to install OS/2 1.1 and I am getting stuck on the ESDI no fixed disks found error.
![Screenshot 2024-04-29 at 10 56 44 PM](https://github.com/86Box/86Box/assets/83831269/fa62d382-16d1-44f1-8f5f-7bf8c5275545)
![Screenshot 2024-04-29 at 10 56 53 PM](https://github.com/86Box/86Box/assets/83831269/ba836e57-0869-43aa-849e-9a80dffe44dc)
![Screenshot 2024-04-29 at 10 59 32 PM](https://github.com/86Box/86Box/assets/83831269/13e10fda-9bd0-49d0-9889-18a0068877be)

**Comment by TC1995** [comment]
Have you used the reference disk first?

**Reply by tifasoftware** [reply]
Yes I have, and did the automatic configuration.

**Reply by tifasoftware** [reply]
I also tried model 80 and it produces the same result.

**Reply by TC1995** [reply]
You need alternative ddff files (@ddff.adf and cddff.adf).

**Reply by tifasoftware** [reply, marked as answer]
The alternative ddff files fixed the issue. However, I had to deal with finding software for patching the floppy so I made a patched floppy and uploaded it to archive.org for other people.
[Patched 70/80 Reference Disk](https://archive.org/details/rf7080a)
